docs/docs-beta/docs/etl-pipeline-tutorial/create-a-sensor-asset.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Create a sensor asset
+description: Use sensors to create event driven pipelines
+last_update:
+  author: Alex Noonan
+sidebar_position: 70
+---
+
+[Sensors](/guides/automate/sensors) in Dagster are a powerful tool for automating workflows based on external events or conditions. They allow you to trigger jobs when specific criteria are met, making them essential for event-driven automation. 
+
+Event driven automations to support situations where jobs occur at irregular cadences or in rapid succession.  is the building block in Dagster you can use to support this. 
+
+Consider using sensors in the following situations:
+- **Event-Driven Workflows**: When your workflow depends on external events, such as the arrival of a new data file or a change in an API response.
+- **Conditional Execution**: When you want to execute jobs only if certain conditions are met, reducing unnecessary computations.
+- **Real-Time Processing**: When you need to process data as soon as it becomes available, rather than waiting for a scheduled time.
+
+In this step you will:
+
+- Create an asset the runs based on a event driven workflow
+- Create a sensor to listen for conditions to materialize the asset. 
+
+## 1. Event Driven Asset
+
+For our pipeline, we want to model a situation where an executive wants a pivot table report of sales results by department and product. They want that processed in real time from their request and it isn't a high priority to build the reporting to have this available and refreshing. 
```

Comment:
I would edit this a bit:
> For our pipeline, we want to model a situation where an executive wants a pivot table report of sales results by department and product, processed at the time of their request.

I think you can leave out the additional info about how the report doesn't need to be continuously available / refreshing in the background.